src/topology/algebra/uniform_group.lean

```diff
@@ -116,8 +126,47 @@ begin
     refine ⟨_, ht, _⟩,
     rintros ⟨a, b⟩, simpa [subset_def] using hts 1 (b / a) a }
 end
+
+@[to_additive] lemma uniformity_eq_comap_nhds_one_swapped :
```

Comment:
Would it make sense to swap the names of `uniformity_eq_comap_nhds_one` and `uniformity_eq_comap_nhds_one_swapped`?

After all `uniformity_eq_comap_nhds_one` is the lemma in which `x.2` appears before `x.1`.

Likewise the lemmas below.

Reply:
I don't think so, `λ x y, y/x` feels to me more natural than `λ x y, x/y`, but I don't have strong feeling about it. Anyways I think it is not in the scope of this PR since it would amount to quite some renaming.